docs/UserGuide.md

```diff
@@ -77,73 +70,254 @@ Format: `help`
 
 ### Adding a person: `add`
 
-Adds a person to the address book.
+Adds a contact to the contact list. A name will be required, and all other information will be optional.
+
+
+Format: `add NAME [-p PHONE_NUMBER] [-e EMAIL] [-s SKILL]...`
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+
+Parameters:
+- `NAME`: Name of the contact. It can be any string except that it cannot start with the hashtag (#) symbol (Reason: To differentiate between indices and names, to be introduced later.), and words cannot start with the hyphen (-) character. (Reason: It will be mixed up with parameter tags.)
```

Comment:
Remove rationale behind decisions, definitely won't be included in the user guide.